bin/ai/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ set(AI_COMPAT_SOURCE_FILES
 	${CMAKE_CURRENT_SOURCE_DIR}/compat_1.10.nut
 	${CMAKE_CURRENT_SOURCE_DIR}/compat_1.11.nut
 	${CMAKE_CURRENT_SOURCE_DIR}/compat_12.nut
+	${CMAKE_CURRENT_SOURCE_DIR}/compat_13.nut
```

Comment:
Not for this PR, but shouldn't this line be added in the 13 branch as well? Same for the bin/game/CMakeLists.txt

Reply:
Well, it's empty for the branch, so doesn't actually do anything.